supported_imus:
   "0"  : { name:  "Unknown",                                      rate:   0   }
   "1"  : { name:  "Honeywell HG1700 AG11",                        rate: 100   }
   "4"  : { name:  "Honeywell HG1700 AG17",                        rate: 100   }
   "5"  : { name:  "Honeywell HG1700 CA29",                        rate: 100   }
   "8"  : { name:  "Northrop Grumman LN-200/LN-200C",              rate: 200   } 
   "11" : { name:  "Honeywell HG1700 AG58",                        rate: 100   } 
   "12" : { name:  "Honeywell HG1700 AG62",                        rate: 100   }
   "13" : { name:  "iMAR iIMU-FSAS",                               rate: 200   }
   "16" : { name:  "KVH CPT IMU",                                  rate: 200   }
   "19" : { name:  "Northrop Grumman Litef LCI-1",                 rate: 200   }
   "20" : { name:  "Honeywell HG1930 AA99",                        rate: 100   }
   "26" : { name:  "Northrop Grumman Litef ISA-100C",              rate: 100   }
   "27" : { name:  "Honeywell HG1900 CA50",                        rate: 100   }
   "28" : { name:  "Honeywell HG1930 CA50",                        rate: 100   }
   "31" : { name:  "Analog Devices ADIS16488",                     rate: 200   }
   "32" : { name:  "Sensonor STIM300",                             rate: 125   }
   "33" : { name:  "KVH1750 IMU",                                  rate: 200   }
   "41" : { name:  "Epson G320N",                                  rate: 125   }
   "45" : { name:  "KVH 1725 IMU",                                 rate: 200   }
   "52" : { name:  "Litef microIMU" ,                              rate: 200   }
   "56" : { name:  "Sensonor STIM300, Direct Connection",          rate: 125   }
   "58" : { name:  "Honeywell HG4930 AN01",                        rate: 200   }
   "61" : { name:  "Epson G370N",                                  rate: 200   }
   "62" : { name:  "Epson G320N - 200Hz",                          rate: 200   }
   "68" : { name:  "HG4930_AN04",                                  rate: 100   }
   "69" : { name:  "HG4930_AN04_400HZ",                            rate: 400   }